content/guides/models/app/features/panels/_index.md

```diff
@@ -34,6 +34,21 @@ To reset a workspace:
 1. At the top of the workspace, click the action menu `...`.
 1. Click **Reset workspace**.
 
+## View a panel in full-screen mode
+
+In full-screen mode:
+
+- A panel plots 10,000 buckets rather than 1,000 when not in full-screen mode.
```

Comment:
```suggestion
- In full-screen, a panel that uses full-fidelity sampling mode plots 10,000 buckets, rather than 1,000 when not in full-screen mode.
```